src/Core/RenderAPI/PipelineOrdering/EspCommandBuffer.hh

```diff
@@ -0,0 +1,21 @@
+#ifndef CORE_RENDER_API_ESP_COMMAND_BUFFER_HH
+#define CORE_RENDER_API_ESP_COMMAND_BUFFER_HH
+
+#include "esppch.hh"
+
+namespace esp
+{
+  class EspCommandBufferId
+  {
+  };
+
+  class EspCommandBuffer
+  {
+    /* -------------------------- STATIC METHODS --------------------------- */
+   public:
+    static std::unique_ptr<EspCommandBufferId> begin_only_once();
```

Comment:
ja bym był za begin single time command

Reply:
ale to nie jest jedna komenda, co nie? tylke jeden cały przebieg renderowania. Po za tym jak chcesz tego użyć to musisz napisać `EspCommandBuffer::begin_only_once()` a tak to musielibyśmy napisać `EspCommandBuffer::begin_single_time_command`, a w sandbox możesz zobaczyć że tam jest więcej niż jedna komenda użyta

Reply:
to może po prostu begin command

Reply:
niech bedzie `begin_commands`, ok?

Reply:
git